client/checkout/woopay/email-input-iframe.js

```diff
@@ -617,13 +616,14 @@ export const handleWooPayEmailInput = async (
 	} );
 
 	if ( ! customerClickedBackButton ) {
-		const paymentMethods = await select(
-			'wc/store/payment'
-		).getAvailablePaymentMethods();
-
-		const hasWCPayPaymentMethod = paymentMethods.hasOwnProperty(
-			'woocommerce_payments'
+		const getWCPayElementOnBlocks = document.getElementById(
```

Comment:
The `get*` prefix could infer that `getWCPayElementOnBlocks` (and `getWCPayElementOnClassic`) are functions, when in reality they're constants.
Maybe `hasWcPayElementOnBlocks` and `hasWcPayElementOnShortcode` could be some alternatives?

Let me know your thoughts!

Reply:
Thank you for reviewing, @frosso!  You are right.  Updated here [a5b4aa0](https://github.com/Automattic/woocommerce-payments/pull/8561/commits/a5b4aa04140fb72b44d6dc1f893f3af04c5bdea3).